index.html

```diff
@@ -22,17 +22,146 @@
+            </nav>
+            <button class="project2">See Project</button>
+        </div>
+
+        <div class="fourth-container">
+            <h3 class="profesional">Profesional Art Printing Data</h3>
+            <p class="par3">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard</p>
+            <nav class="langs2">
+                <button class="langbuttons2">html</button>
+                <button class="langbuttons2">bootstrap</button>
+                <button class="langbuttons2">Ruby</button>
+            </nav>
+            <button class="project2">See Project</button>
+        </div>
+
+        <div class="fourth-container">
+            <h3 class="profesional">Profesional Art Printing Data</h3>
+            <p class="par3">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard</p>
+            <nav class="langs2">
+                <button class="langbuttons2">html</button>
+                <button class="langbuttons2">bootstrap</button>
+                <button class="langbuttons2">Ruby</button>
+            </nav>
+            <button class="project2">See Project</button>
+        </div>
+
+        <div class="fourth-container">
+            <h3 class="profesional">Profesional Art Printing Data</h3>
+            <p class="par3">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard</p>
+            <nav class="langs2">
+                <button class="langbuttons2">html</button>
+                <button class="langbuttons2">bootstrap</button>
+                <button class="langbuttons2">Ruby</button>
+            </nav>
+            <button class="project2">See Project</button>
+        </div>
+
+        <div class="fourth-container">
+            <h3 class="profesional">Profesional Art Printing Data</h3>
+            <p class="par3">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard</p>
+            <nav class="langs2">
+                <button class="langbuttons2">html</button>
+                <button class="langbuttons2">bootstrap</button>
+                <button class="langbuttons2">Ruby</button>
+            </nav>
+            <button class="project2">See Project</button>
+        </div>
+
+        <div class="fourth-container">
+            <h3 class="profesional">Profesional Art Printing Data</h3>
+            <p class="par3">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard</p>
+            <nav class="langs2">
+                <button class="langbuttons2">html</button>
+                <button class="langbuttons2">bootstrap</button>
+                <button class="langbuttons2">Ruby</button>
+            </nav>
+            <button class="project2">See Project</button>
+        </div>
+
```

Comment:
Next, let's wrap all the project cards named `.fourth-container` into their own container. For illustration purposes, I'll name it `fourth-container-wrapper`:
![image](https://user-images.githubusercontent.com/38088818/164783498-4793c443-b0ec-44fb-ba30-3d3c18a6689e.png)

Now, let's use our `fourth-container-wrapper` to layout our `.fourth-container` child elements:
If I apply a `margin` of zero and a 100% `width`. The`red` and `green` borders are for illustration purposes.
![image](https://user-images.githubusercontent.com/38088818/164784150-481d8644-d46f-4a55-a163-0f27f01ef245.png)

The structure will look like this for now:
![image](https://user-images.githubusercontent.com/38088818/164784534-0a05d6ab-c28e-4450-b91e-e2b1f6d3db22.png)


Then I apply Flex properties and add `margin` zero on the `.fourth-container` as well:
![image](https://user-images.githubusercontent.com/38088818/164785220-acf6180d-8805-4d1f-b22f-291de1a294e4.png)

We now have this:
![image](https://user-images.githubusercontent.com/38088818/164785617-f782e634-8268-4d41-9b78-a532bf4789c3.png)

All these steps are of course my take on how I would go about resolving your display issues. You are more than welcome to do it in your own way and understanding. These are merely my suggestions, however, the final product should be the same: which is making sure that your app looks as close to the Figma design as possible.

Good luck and happy coding 🥂

Reply:
About the fourth-container-wrapper, I did not implement it because when I checked the page it was already centered. I think this is because the corrections made in step 2. If you feel I still should do it please let me know it won't be a problem.

Reply:
That's alright, I did mention that some of the listed suggestions are merely my take on structuring your elements. With the exception of grouping your sections in `section` tag and using a `ul` tag for listing technologies; which are part of the project requirements. You can do it in your own way and understanding 💯